flash/image/segmentation/input.py

```diff
@@ -33,7 +32,6 @@
     SampleCollection = None
 
 if _TORCHVISION_AVAILABLE:
-    import torchvision
     import torchvision.transforms.functional as FT
```

Comment:
```suggestion
    import torchvision.transforms.functional as TVF
```

This might be more readable

Reply:
Changed to just import `to_tensor` as it's the only one we use :smiley: